src/components/Tag/__stories__/index.stories.tsx

```diff
@@ -1,33 +1,83 @@
 import { Meta, Story } from '@storybook/react'
 import { ComponentProps } from 'react'
 import Tag from '..'
+import { SENTIMENTS } from '../../../theme'
 
 export default {
-  args: {
-    children: 'default',
+  component: Tag,
+  title: 'Components/Data Display/Tag',
+} as Meta
+
+const Template: Story<ComponentProps<typeof Tag>> = args => (
+  <Tag {...args}>Tag</Tag>
+)
+
+export const Default = Template.bind({})
+
+export const Variants = Template.bind({})
+Variants.parameters = {
+  docs: {
+    description: {
+      story:
+        'Variants defines different colors of you component. You can define it using `variant` property.',
+    },
   },
-  argTypes: {
-    children: {
-      defaultValue: 'This is a children',
-      name: 'children',
-      table: {
-        defaultValue: { summary: 'This is a children' },
-        type: { required: true, summary: 'string' },
-      },
-      type: { name: 'string', required: true },
+}
+Variants.decorators = [
+  () => (
+    <div style={{ display: 'flex', flexWrap: 'wrap', gap: 8 }}>
+      {SENTIMENTS.map(sentiment => (
+        <Tag key={sentiment} variant={sentiment} onClose={() => {}}>
+          {sentiment}
+        </Tag>
+      ))}
+    </div>
+  ),
+]
+
+export const Icons = Template.bind({})
+Icons.parameters = {
+  docs: {
+    description: {
+      story:
+        'Add Icon on left side of your tag, You can define it using `icon` property.',
     },
   },
-  component: Tag,
-  parameters: {
-    docs: {
-      description: {
-        component: 'Compact element to display information inside a box.',
-      },
+}
+Icons.args = {
+  icon: 'check',
+}
+
+export const onClose = Template.bind({})
+onClose.parameters = {
+  docs: {
+    description: {
+      story:
+        '`onClose` property to pass function trigger on click on close button.',
     },
   },
-  title: 'Components/Data Display/Tag',
-} as Meta
+}
+onClose.args = {
+  icon: 'check',
+  onClose: () => {},
+}
```

Comment:
```suggestion
onClose.args = {
  onClose: () => {},
}
```

Lets just show the prop linked to title